recipes/toolchain/deactivate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
You would be surprised the things I see people doing with the `gcc` package. A little safety didn't hurt anyone. 😉